ostd/src/mm/vm_space.rs

```diff
@@ -396,16 +384,11 @@ impl<'a> CursorMut<'a> {
                         self.flusher
                             .issue_tlb_flush_with(TlbFlushOp::Address(start_va), old_frame.into());
                     }
-                    MappedItem::ReservedFrame(old_frame) => {
-                        self.flusher
-                            .issue_tlb_flush_with(TlbFlushOp::Address(start_va), old_frame.into());
-                    }
                     MappedItem::UntrackedIoMem { .. } => {
-                        // Flush the TLB entry for the current address, but in
-                        // the current design, we cannot drop the corresponding
-                        // `IoMem`. This is because we manage the range of I/O
-                        // as a whole, but the frames handled here might be one
-                        // segment of it.
+                        // Flush the TLB entry for the current address, but DO
+                        // NOT drop the corresponding `IoMem`. This is because
+                        // we manage the range of I/O as a whole, but the
+                        // frames handled here might be one segment of it.
```

Comment:
Be more careful in rebasing the changes? I think you've updated the comments but you restore the comments to the old version...